src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -422,7 +422,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 			fastScrollModifier: 'alt',
 			fastScrollSensitivity: editorOptions.fastScrollSensitivity,
 			scrollSensitivity: editorOptions.mouseWheelScrollSensitivity,
-			rendererType: config.rendererType === 'auto' || config.rendererType === 'experimentalWebgl' ? 'dom' : config.rendererType,
+			rendererType: config.rendererType === 'auto' || config.rendererType === 'experimentalWebgl' ? 'canvas' : config.rendererType,
```

Comment:
Let's use brackets around the first part, I'm not 100% sure whether the execution order is:

`(a || b) ? c : d` or `a || (b ? c : d)`